frontend/src/features/crawl-workflows/templates/shareable-notice.ts

```diff
@@ -0,0 +1,14 @@
+import { msg } from "@lit/localize";
+import { html } from "lit";
+
+export const ShareableNotice = () =>
+  html`<btrix-popover
+    content=${msg(
+      "The latest crawl from this workflow is publicly accessible to anyone with the link. This can be changed with the Browsertrix API.",
+    )}
+  >
+    <btrix-badge class="min-h-5" variant="warning">
```

Comment:
Any reason not to use `part-[base]`?
```suggestion
    <btrix-badge class="part-[base]:min-h-5" variant="warning">
```

Reply:
Oh, right, yeah I think that's nicer. I guess it does feel a bit like you shouldn't _have_ to reach into the internals of a component to apply styling that seems like it should affect the component as a whole, but this is what we do elsewhere, so it makes sense not to change that now.